src/components/HeadCellMenu/HeadCellMenu.tsx

```diff
@@ -72,11 +71,7 @@ const HeadCellMenu = ({
   if (!interactions.active) return null;
 
   return (
-    <HeadCellMenuWrapper
-      tabIndex={tabIndex}
-      rightAligned={headTextAlign === 'right'}
-      shouldShowMenuIcon={shouldShowMenuIcon}
-    >
+    <HeadCellMenuWrapper rightAligned={headTextAlign === 'right'} shouldShowMenuIcon={shouldShowMenuIcon}>
```

Comment:
without the property, the default is -1?

Reply:
not actually, no tabIndex -> no stop at all, like any other element
-1 only would means that it **might be ignored (specifically)** within the tab stop flow.

Technically, the presence of tabIndex would mean a stop, and -1 is being ignored within those element that has tab stops, and an element having no tab stop, would means that it's not dealing with tab stop at all. Just being a neutral element sitting in DOM somewhere.

Reply:
huh, didn't know that, good to know